### ISSUE TITLE
Accessibility Bug: There is no text alternative in the form of tooltip provided for the highlighted button in the shell bar region

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3145]
Fundamental core component: "Product switch". 

Description:  There is no text alternative in the form of tooltip provided for the highlighted button in the shell bar region

Expected: The text alternative should be provided for every non-text content element which fulfills the purpose of the content.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91651531-7f0bac00-eaab-11ea-808e-3facdb89d9b9.png)
